packages/core/src/index.ts

```diff
@@ -74,10 +74,19 @@ import { CustomStyleRegistry } from './registries/CustomStyleRegistry.js';
 import { NativeStyleRegistry } from './registries/NativeStyleRegistry.js';
 import { global } from './utils/global.js';
 
-global.elf = global.Elf = global.ELF = {
-  customStyles: CustomStyleRegistry,
-  nativeStyles: NativeStyleRegistry,
-  version: 'PUBLISH_VERSION'
-};
+interface EfStylesDefine extends CustomEvent {
+  detail: {
+    name: string,
+    styles: string
+  }
+}
+
+global.addEventListener('ef.customStyles.define', (event) => {
+  const { name, styles } = (event as EfStylesDefine).detail;
+  CustomStyleRegistry.define(name, styles);
+});
 
-Object.freeze(global.elf);
+global.addEventListener('ef.nativeStyles.define', (event) => {
+  const { name, styles } = (event as EfStylesDefine).detail;
+  NativeStyleRegistry.define(name, styles);
+});
```

Comment:
Don't worry about interface, just cast to string inside of the event listener. This section is temporary and will be removed soon.

Reply:
I think this cast cannot turn to be string. The purpose of this cast uses for eslint.

Reply:
Updated.